src/java/za/co/mip/ablduck/javadoc/JavadocParserListener.java

```diff
@@ -0,0 +1,230 @@
+// Generated from /home/roberte/Projects/pct/src/java/za/co/mip/ablduck/javadoc/JavadocParser.g4 by ANTLR 4.7
+package za.co.mip.ablduck;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "JavadocParserListener.java" should be located in "za/co/mip/ablduck" directory, not in "/ebs/ebs1/jenkins/workspace/PCTNewJob_PR-247-X33Q424TCA4ZMZR72S4Y6PBXSKT7FL6QYGBFILF26W7CMTBRNBCA/src/java/za/co/mip/ablduck/javadoc". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1598)